core_ui/lib/src/widgets/main_screen/mobile/report_statistics.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:api_client/api_client.dart';
+import 'package:flutter/material.dart';
+
+class ReportStatistics extends StatelessWidget {
+  const ReportStatistics({
+    super.key,
+    required this.width,
+    required this.reportStatistics,
+    required this.isMobile,
+  });
+
+  final double width;
+  final ReportStatisticsDto reportStatistics;
+  final bool isMobile;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: width,
+      child: isMobile
+          ? Column(children: getStatWidgets(reportStatistics))
+          : Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: getStatWidgets(reportStatistics)),
```

Comment:
To enhance the mobile viewing experience and eliminate the need for explicit width settings or differentiation between mobile and desktop, leverage the `Wrap` widget. This widget, configured with appropriate alignment and spacing parameters, ensures that reports are presented more aesthetically on mobile devices. The example below demonstrates the implementation:

```dart
return Wrap(
  alignment: WrapAlignment.center,
  spacing: 8,
  runSpacing: 4,
  children: [
    ReportTypeStatistics(
      title: 'Gauti',
      total: reportStatistics.totalSentReports,
      imageShort: 'red',
      tooltipText:
          'Pranešimą peržiūrėjo tvarkaulietuva.lt administratorius ir jis perduotas atitinkamam struktūriniam vienetui tvarkyti.',
    )
  ],
);
```

This approach enhances responsiveness, ensuring that reports are visually appealing on various screen sizes. 

<img width="400" alt="Screenshot 2024-01-18 at 08 20 46" src="https://github.com/AplinkosMinisterija/tvarkau-lietuva-frontend/assets/3719141/920f8cc7-a4a0-49df-ba0b-c2128d9fa0ea">